lib/kernel/src/group.erl

```diff
@@ -652,8 +652,28 @@ get_line1({search,Cs,Cont,Rs}, Drv, Shell, Ls, Encoding) ->
     %% prompt ('N>') and substitute it with the search prompt.
     put(search_quit_prompt, Cont),
     Pbs = prompt_bytes("\033[;1;4msearch:\033[0m ", Encoding),
-    {more_chars,Ncont,_Nrs} = edlin:start(Pbs, {search,none}),
+    send_drv_reqs(Drv, edlin:erase_line()),
+    {more_chars,Ncont,Nrs} = edlin:start(Pbs, {search,none}),
+    send_drv_reqs(Drv, Nrs),
     get_line1(edlin:edit_line1(Cs, Ncont), Drv, Shell, Ls, Encoding);
+get_line1({help, Before, Cs0, Cont, Rs}, Drv, Shell, Ls0, Encoding) ->
+    send_drv_reqs(Drv, Rs),
+    {_,Word,_} = edlin:over_word(Before, [], 0),
+    Docs = case edlin_context:get_context(Before) of
+        {function, Mod} when Word =/= [] -> try
+                    c:h1(list_to_atom(Mod), list_to_atom(Word))
```

Comment:
What happens if `Mod` is something like `'OTP-PUB-KEY'`. If you use `erl_scan:string/1` it will handle the `'` for you.